Fixed that miss colored '1' button on the safe and nuke keypad

WarlockD opened the pull request:
## About The Pull Request

Its been driving me crazy.  So I fixed it.  I also went a head and set the wall/captin safe to the "retro" template like the nuke button is so all the shadows come in right.  While I was at it I removed the obsoleted Grid control and just did it all with Flex.
## Why It's Good For The Game

The keypad is more consistent.
![2pxgJs0id3](https://github.com/tgstation/tgstation/assets/1102671/00ca2b9d-ba9f-4534-809d-d19bb40ae89a) | ![nBIAozJxok](https://github.com/tgstation/tgstation/assets/1102671/75bf9597-a396-455b-9157-28fb1bcbe856)
## Changelog
:cl:
fix: The '1' button is the right color
fix: Backgrounds look better on normal safes
:cl:

WarlockD committed:
Fixed up the Keypad

WarlockD committed:
The status text was just a bit long

I am might end up making a new pad at some point

MrMelbert commented on tgui/packages/tgui/styles/interfaces/NuclearBomb.scss:
```diff
@@ -32,12 +32,6 @@ $background-beige: #e8e4c9;
   }
 }
 
-.NuclearBomb__Button--1 {
-  background-color: #d3cfb7 !important;
-  border-color: #d3cfb7 !important;
-  color: #a9a692 !important;
-}
```
It's intentionally discolored, that's why it's a different class

MrMelbert replied:
![image](https://github.com/tgstation/tgstation/assets/51863163/b5a6a178-153b-45d1-aafb-9efd037126f2)

WarlockD replied:
Humm see I was wondering about why its been like that for so long then or why it had its own class. I will set it up so you can have the main device that uses the pad send classes for it instead.  Humm  kind of makes me wonder if I can put random "smudges" or fingerprints on some of the keys.  Let me play with it.

WarlockD replied:
![i5hpreOPp6](https://github.com/tgstation/tgstation/assets/1102671/1227613a-2319-495e-a82f-f3e4d718b340)
There we go, the normal safe keypad is on a separate css class.

JohnFulpWillard commented:
This was all intentional so you need to justify why we should change the intended look, not that is a fix

DingleFlop commented:
Perhaps adding a hover effect, instead of having a single, statically discolored button, would be a reasonable compromise. The best of both worlds, even.

WarlockD committed:
Updated NukeBomb to ts

Fixed the number 1 keypad issue

WarlockD commented:
Ok did some minor updates.  Just changed the NukeBomb to be Typescript, moved it to Flex, then set each on their own class.  With 516 on the way I didn't want to use ms-grid to fix all the alignment issuse.
